roles/pg_replicant/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+# based on https://github.com/lesovsky/ansible-postgresql-sr-on-el6
+
+- name: stop postgresql on replicant
+  service:
+    name: postgresql
+    state: stopped
+
+- name: remove old postgresql cluster on replicant
+  file:
+    path: XXX
+    state: absent
+
+# pg_basebackup
+
+- name: configure recovery.conf
+  template:
+    src: XXX/recovery.conf
+    dest: "{{ postgresql_data_dir }}/recovery.conf"
```

Comment:
That's a good fit w/ debian standard deployment, and is flexible for other installs.